08_files_lab/02.py

```diff
@@ -1,7 +1,27 @@
-"""
-Write a program that takes 3 names. The first two
-are existing file names and the last is a new file name
-Program should write the lines from the first two 
-files interwinded into the output file
-"""
-
+import sys 
+import itertools
+
+(_, file1, file2, file3) = sys.argv
+
+def write():
+    new_file = (raw_input("./" + file3))
+
+from itertools import izip_longest
+with open ("./" + file1, 'r') as text1:
+    with open ("./" + file2, 'r') as text2:
+        with open ("./" + file3, 'w') as text3:
+            for line1, line2 in izip_longest(text1, text2, fillvalue = ""):
+                text3.write(line1 + line2)
+
+
+
+"""" my ootput is : 
+one
+red
+two
+blue
+threegreen
+white
+orange
+
+how can i split the str "threegreen" ? """"
```

Comment:
What does the ```write``` function do here?
You need to add a newline after printing "three", but what is the general case of this problem?